Cargo.toml

```diff
@@ -11,3 +11,14 @@ include = ["lib.rs", "utils.rs", "README.md", "LICENSE-APACHE", "LICENSE-MIT"]
 description = "near-gas is crate for work with gas data in near-protocol."
 
 [dependencies]
+serde = { version = "1", features = ["derive"], optional = true }
+borsh = { version = "0.10.3", features = ["const-generics"], optional = true }
+schemars = { version = "0.8.8", optional = true }
```

Comment:
Please, document these as features in the doc-comments, so users of the crate can easily find what they need. See https://docs.rs/borsh/1.0.0-alpha.4/borsh/index.html as a role model.